README.md

```diff
@@ -244,6 +252,60 @@ This is useful on development machines. It allows you to:
 
 All while running Vaultenv without any CLI args.
 
+It can happen that conflicting configuration values are send to Vaultenv. An example
```

Comment:
I would have separate paragraphs for the fact ("Vaultenv supports different levels of config. They have an order of precedence") with the use-case ("For example, machine specific config, project specific config")